package.json

```
@@ -3,6 +3,8 @@
   "description": "Safety wrapper around the Web Animation API",
   "repository": "choojs/nanoanimation",
   "version": "2.1.0",
+  "main": "index.js",
+  "module": "index.es.js",
```

```suggestion
  "module": "index.mjs",
```

Maybe we should change this to be a `.mjs` file instead? I believe that's where Node + all the bundlers are heading, and it would allow us to be compatible out of the box.

---

> I believe that's where Node + all the bundlers are heading

Yeah, but not browsers, one of the requirements for es modules is the javascript header, and browsers (only checked in chrome actually) doesn't seem to bind javascript header with mjs extension.

---

the content-type header is the thing that matters to browsers, not the extension. usually with .mjs the problem is that the server doesn't send the correct headers for the file type, not everything has that mapping built in yet.

---

The current ideal path for ES modules with Node is having an extensionless `main`:
```json
"main": "index",
"module": "index.mjs"
```
because Node doesn't use the `module` property (I'm not sure if it will in the future). Things like webpack and rollup can pick up the `module` field, and Node (+ browserify in the future) will try to use `$main + .mjs` when import syntax was used.

---

Ok, I agree with this change :)